domain/domain.go

```diff
@@ -2110,7 +2111,9 @@ func (do *Domain) updateStatsWorker(ctx sessionctx.Context, owner owner.Manager)
 			}
 
 		case <-readMemTricker.C:
-			memory.ForceReadMemStats()
+			if gctuner.EnableGOGCTuner.Load() {
```

Comment:
Not only GOGCTuner needs it, but All also need it to upgrade memstat to avoid frequence STW.

Reply:
Now we read the memstat by ```memory.ReadMemStats()```